src/devices/Adxl345/Adxl345.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+using System.Numerics;
+
+namespace Iot.Device.Adxl345
+{
+    /// <summary>
+    /// SPI Accelerometer ADX1345
+    /// </summary>
+    public class Adxl345 : IDisposable
+    {
+        private SpiDevice _sensor = null;
+                             
+        private readonly byte _gravityRangeByte;                             
+        private readonly int _range;
+
+        private const int Resolution = 1024;        // All g ranges resolution
+
+        #region SpiSetting
+        /// <summary>
+        /// ADX1345 SPI Clock Frequency
+        /// </summary>
+        public const int SpiClockFrequency = 5000000;
+
+        /// <summary>
+        /// ADX1345 SPI Mode
+        /// </summary>
+        public const SpiMode SpiMode = System.Device.Spi.SpiMode.Mode3;
```

Comment:
Wow so is this device doesn't use Mode0 as the default? I'm curious, Is this documented on the datasheet?

Reply:
@joperezr you can see SPI part in page 15(README has datasheet link)😁

Reply:
First one I've seen :smile: